index.ts

```diff
@@ -70,7 +70,7 @@ const isbnResolver = new IsbnResolver();
 
 async function handleBotCommand(msgText: string, files: any[], userID: string, downloadToken: string): Promise<SlackMessage> {
     // strip the <@USERID> app mention
-    msgText = msgText.replace(/<@.*> /g, "");
+    msgText = msgText.replace(/^<@.*> ?/, "");
```

Comment:
Will the bot ever get a message without the mention at the front?

Reply:
No valid messages.